### DISCUSSION TITLE
List of backend services to use with Next.js

### DISCUSSION THREAD

**Discussion by leerob** [vercel/next.js#16141] [Show and tell]
Hey everyone! As a front-end developer who stumbled into figuring out the back-end, I wanted to create a resource to help others figure out which backend to use for their Next.js (or other front-end) apps. 

[You can check out the full post here.](https://leerob.io/blog/backend) Here's a high-level summary:

- Databases: Postgres, Mongo, Fauna, Dynamo, etc.
- APIs (auto-generated and custom): Hasura, Nhost, AppSync, Nexus, Apollo.
- All-in-one services: Firebase, Userbase, Supabase, Amplify.
- DB clients/ORMs, which includes Prisma, TypeORM, etc.

It's worth mentioning, for Next.js specifically, you can use API routes in conjunction with some of these services, depending on how much of the backend you want to build yourself.

**Comment by Vadorequest** [comment]
@leerob Totally agree on using some low-code solution, such as Airtable.

That's actually what I decided to use with NRN boilerplate, I was using GraphQL (GraphCMS vendor) but figured using something light like Airtable is actually much easier to manage, not only as a developer, but also more end-user friendly.

NRN repo: https://github.com/UnlyEd/next-right-now
NRN doc about the Airtable preset: https://unlyed.github.io/next-right-now/available-presets/v2-mst-aptd-at-lcz-sty
